src/main/java/org/jenkinsci/plugins/github/admin/GitHubDuplicateEventsMonitor.java

```diff
@@ -206,10 +207,10 @@ public static TrackedDuplicateEvent getLastDuplicate() {
          */
         @VisibleForTesting
         @Restricted(NoExternalUse.class)
-        static Set<String> getEventCountsTracker() {
-            return EVENT_TRACKER.asMap().keySet().stream()
-                                .filter(key -> EVENT_TRACKER.getIfPresent(key) != null)
-                                .collect(Collectors.toSet());
+        Set<String> getEventCountsTracker() {
+            return eventTracker.asMap().keySet().stream()
+                               .filter(key -> eventTracker.getIfPresent(key) != null)
```

Comment:
BTW I think this would be better written using a stream on `entrySet`.

Reply:
`entrySet` because it is more performance optimal?

would be like,
```
return eventTracker.asMap().entrySet().stream()
                   .filter(entry -> entry.getValue() != null)
                   .map(Map.Entry::getKey)
                   .collect(Collectors.toSet());
```

But since this is a helper method for tests only, and we only need the keys, ig probably keeping on `keySet()` is easy read.. 🤔 

(I now renamed the method to `getPresentEventKeys` which makes more sense)

Reply:
Yeah it is just a bad pattern to iterate a map’s key set and then look up the value each time. Does not really matter of course.